olGO/Models/User.swift

```diff
@@ -23,17 +24,19 @@ struct SocialInformation: Codable {
     var location: String = ""
 }
 
-struct User: Codable {
+final class User: SQLiteModel, Codable {
```

Comment:
What else do we want to store in memory sqlite? Is it worth it?

Reply:
I was debating if we should store the user... Only other things that might be nice to store are Posts...

Also I'd probably switch `memory` to a `file` to cache values on device and not use UserDefaults or CoreData...